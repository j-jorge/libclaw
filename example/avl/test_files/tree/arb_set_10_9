 46 46 86 85 19 89 15 11 50 71 15