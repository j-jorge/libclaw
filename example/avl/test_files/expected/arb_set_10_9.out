Tree is empty
File loading...
size : 9
Min = 11
Max = 89
Tree's not empty
Found : 11 15 19 46 50 71 85 86 89
Scan with an iterator : 11 15 19 46 50 71 85 86 89
Scan bis, iterator affectation :11: 11 15 19 46 50 71 85 86 89
15: 15 19 46 50 71 85 86 89
19: 19 46 50 71 85 86 89
46: 46 50 71 85 86 89
50: 50 71 85 86 89
71: 71 85 86 89
85: 85 86 89
86: 86 89
89: 89
Deletions
Copy and scan
source size : 0
copy size : 0
Tree's copy is empty

Reverse scan

Assignment
Clear
Tree's is empty
